types/chart.js/index.d.ts

```diff
@@ -195,6 +195,7 @@ declare namespace Chart {
         events?: string[];
         onHover?(this: Chart, event: MouseEvent, activeElements: Array<{}>): any;
         onClick?(event?: MouseEvent, activeElements?: Array<{}>): any;
+		legendCallback?(chart: any): any;
```

Comment:
According to docs it should return `string`. Not sure it's necessary to annotate it as `string` but it could be useful to prevent `[object Object]` kind of errors

Reply:
@chicoxyzzy Do you think I should change my solution ? If you confirm that the type is a 'string', I may change.

Reply:
it coerces to string anyway but yes, I'd prefer if it will be explicitly `string`